js/ctrls/modal.js

```diff
@@ -37,7 +37,7 @@ angular
     open: function(cb) {
       var self = this;
       this.uris = "";
-      this.downloadSettingsCollapsed = true;
+      this.downloadSettingsCollapsed = false;
```

Comment:
@ipcjs was it done on purpose or accidentally? Doesn't seem related to the issue. The idea of #298 was to have all settings collapsed by default to save vertical space, and now it's broken.

/cc @ziahamza

Reply:
doesnt seem to be related to this issue, I will revert it back to `true` @morozov !